docker-compose.yml

```diff
@@ -90,53 +90,36 @@ services:
       - APP_NAME=avalon
       - AVALON_BRANCH=develop
       - BUNDLE_FLAGS=--with development postgres --without production test
+      - ENCODE_WORK_DIR=/streamfiles
       - SETTINGS__DOMAIN=http://localhost:3000
       - CONTROLLED_VOCABULARY=config/controlled_vocabulary.yml
       - DATABASE_URL=postgres://postgres@db/avalon
       - SETTINGS__DROPBOX__PATH=/masterfiles
       - SETTINGS__DROPBOX__UPLOAD_URI=./masterfiles
-      - EMAIL_COMMENTS
-      - EMAIL_NOTIFICATION
-      - EMAIL_SUPPORT
-      - ENCODE_WORK_DIR=/streamfiles
-      - FEDORA_BASE_PATH
       - FEDORA_NAMESPACE=avalon
       - FEDORA_URL=http://fedoraAdmin:fedoraAdmin@fedora:8080/fedora/rest
       - SETTINGS__FFMPEG__PATH=/usr/bin/ffmpeg
-      - MASTER_FILE_PATH
-      - MASTER_FILE_STRATEGY=delete
+      - MASTER_FILE_STRATEGY=none
```

Comment:
Does it make sense to default this to `none` or should it get set back to `delete`?

Reply:
In development I found it easier to debug if masterfiles aren't deleted. These are visible outside of the container and can be easily managed if necessary.

Reply:
But this docker-compose.yml is being used for more than just development, right?  If it is then I think we should try to mirror best practices and not what might be best for specific development tasks.  Is this a change that you can just make locally for now?

Reply:
This docker-compose is just used for development. Actually I'll just get rid of it and use the default settings.yml.